github/Enterprise.py

```diff
@@ -0,0 +1,66 @@
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2023 Yugo Hino <henom06@gmail.com>                                 #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+from typing import Any, Dict
+
+from github.EnterpriseConsumedLicenses import EnterpriseConsumedLicenses
+from github.GithubObject import Attribute, CompletableGithubObject, NotSet
+
+
+class Enterprise(CompletableGithubObject):
+    """
+    This class represents Enterprises. The reference can be found here https://docs.github.com/en/enterprise-cloud@latest/rest/enterprise-admin/license#list-enterprise-consumed-licenses
+    """
+
+    def _initAttributes(self) -> None:
+        self._login: Attribute[str] = NotSet
```

Comment:
Docs call he `enterprise` parameter

> The slug version of the enterprise name. You can also substitute this value with the enterprise id.

https://docs.github.com/en/enterprise-cloud@latest/rest/enterprise-admin/license?apiVersion=2022-11-28#list-enterprise-consumed-licenses

So I'd rather go for `slug_or_id`.

Reply:
Thank you. Fixed it.